templates/documentation/delivery-analytics/private-video-on-hls-or-external-players.md

```diff
@@ -1,11 +1,11 @@
 ---
-title: "Private video delivery with HLS built-in players"
-slug: "private-video-on-hls-or-external-players"
+title: Private video delivery with HLS built-in players
+slug: private-video-on-hls-or-external-players
```

Comment:
`slug` can be removed, as confirmed by Doctave [here](https://api-video.slack.com/archives/C05DTNR3QDS/p1697618392282579)

Reply:
Done ✅